rbx_binary/src/serializer/state.rs

```diff
@@ -709,6 +709,21 @@ impl<'dom, W: Write> SerializerState<'dom, W> {
                         chunk.write_interleaved_i32_array(offset_x.into_iter())?;
                         chunk.write_interleaved_i32_array(offset_y.into_iter())?;
                     }
+                    Type::Font => {
+                        for (i, rbx_value) in values {
+                            if let Variant::Font(value) = rbx_value.as_ref() {
+                                chunk.write_string(&value.family)?;
+                                chunk.write_le_u16(value.weight.clone().into())?;
+                                chunk.write_u8(value.style.clone().into())?;
+                                match &value.cached_face_id {
+                                    Some(id) => chunk.write_string(id)?,
+                                    None => chunk.write_string("")?,
+                                }
```

Comment:
Nit: Would this look nicer?

```suggestion
                                chunk.write_string(value.cached_face_id.unwrap_or_default())?;
```

Reply:
I think so. I had to stick a `.clone()` to make the borrow checker happy, and write_string wants an `&str`, so it's a bit uglier than your suggestion but I think it's still nicer than what I had.